features/order/managing_orders/seeing_order_item_detailed_data.feature

```diff
@@ -25,7 +25,8 @@ Feature: Seeing order item detailed data
     Scenario: Seeing details of item in one row
         Given I view the summary of the order "#00000666"
         When I check "Iron Man T-Shirt" data
-        Then its unit price should be $39.00
+        Then its code should be "IRON_MAN_T-SHIRT"
```

Comment:
To verify this fix is correct we should add two variants of the same product and validate their codes.